### PR TITLE
[onert] Argmax acl cl int64

### PR DESCRIPTION
**Pull request by dr-venkman**
Support int64 output type for Argmax on acl/cl backend.

ONE-DCO-1.0-Signed-off-by: venkat.iyer <venkat.iyer@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
Please add unittest.

**Comment by dr-venkman** [comment]
I tried extending the testcase to handle a new `TENSOR_INT64` type. However, it fails to generate a suitable `argmax_float_x_nnfw.example.cpp` upon compilation. Adding int64 testcases seems a non-trivial task. Could you please let me know if there are testcases existing already that deal with int64? Alternatively, would it help to open an issue and reference it here?

**Comment by hseok-oh** [comment]
@dr-venkman Use `one_op_tests` (`tests/nnfw_api/src/one_op_tests`), not nnapi test. NNAPI don't have INT64 type. There is no INT64 type example, but you can extend type easily than NNAPI.

**Comment by dr-venkman** [comment]
> @dr-venkman Use `one_op_tests` (`tests/nnfw_api/src/one_op_tests`), not nnapi test. NNAPI don't have INT64 type. There is no INT64 type example yet, but you can extend type easily than NNAPI.

@hseok-oh , Thanks, PTAL.

File: tests/nnfw_api/src/one_op_tests/ArgMax.cc [hunk 1]
```diff
@@ -40,6 +40,28 @@ TEST_F(GenModelTest, OneOp_ArgMax_AxisToConst)
   SUCCEED();
 }
 
+TEST_F(GenModelTest, OneOp_ArgMax64_AxisToConst)
```
Comment (hseok-oh) [inline tests/nnfw_api/src/one_op_tests/ArgMax.cc:43]: Please change name `OneOp_ArgMax64_AxisToConst` to `OneOp_ArgMax_Int64_AxisToConst`

**Comment by hseok-oh** [comment]
@ragmani Could you check this PR?

File: compute/ARMComputeEx/src/runtime/CL/functions/CLArgMinMaxLayerEx.cpp [hunk 2]
```diff
@@ -0,0 +1,205 @@
+/*
```
Comment (intom) [inline compute/ARMComputeEx/src/runtime/CL/functions/CLArgMinMaxLayerEx.cpp:1]: please add copyright

File: tests/nnfw_api/src/one_op_tests/ArgMax.cc (hunk 1 above)
Comment (dr-venkman) [reply tests/nnfw_api/src/one_op_tests/ArgMax.cc]: Thanks for the comment, I have modified accordingly.

File: compute/ARMComputeEx/src/runtime/CL/functions/CLArgMinMaxLayerEx.cpp (hunk 2 above)
Comment (dr-venkman) [reply compute/ARMComputeEx/src/runtime/CL/functions/CLArgMinMaxLayerEx.cpp:1]: Added to all modified files under armcomputeex, kindly review. Thanks.

**Review by hseok-oh — APPROVED** [review]

**Review by intom — APPROVED** [review]
LGTM

**Pull request merged**